rapids-cmake/cpm/versions.json

```diff
@@ -23,6 +23,13 @@
       "git_url": "https://github.com/NVIDIA/cuCollections.git",
       "git_tag": "096346b739da3fb1d9c3b402190c0a3a7e554440"
     },
+    "rapids_logger": {
+      "version": "0.1.0",
+      "git_shallow": false,
+      "always_download": true,
+      "git_url": "https://github.com/rapidsai/rapids-logger",
+      "git_tag": "98cb4d232dc4612f80d462a0a7bc5490dbc22399"
```

Comment:
Will this be versioned like RAPIDS (25.02) or SemVer or something else? It would be nice if this used proper tags instead of commit hashes.

Reply:
I would like for it to use SemVer, but I'm not married to that. In general I would expect it to change pretty infrequently once we're happy with its state, so I don't think matching RAPIDS CalVer would be that useful since we'd be making a lot of identical releases.

However, I definitely don't want to address rapids-logger versioning quite yet. The package is changing too frequently and we'll wind up bumping versions very often. Let's not hold up this PR on the versioning decision.

Reply:
That's fine, I also like SemVer for this. Before 25.02 goes out, I'd like this to be a tag instead of a hash.

Reply:
Yes, it's always been my intent to reach a 0.1 release or equivalent for rapids-logger for RAPIDS 25.02.